modules/core/include/opencv2/core/mat.hpp

```diff
@@ -183,7 +183,8 @@ class CV_EXPORTS _InputArray
 #if OPENCV_ABI_COMPATIBILITY < 500
         STD_ARRAY         =14 << KIND_SHIFT,  //!< removed: https://github.com/opencv/opencv/issues/18897
 #endif
-        STD_ARRAY_MAT     =15 << KIND_SHIFT
+        STD_ARRAY_MAT     =15 << KIND_SHIFT,
+        NPU_MAT           =16 << KIND_SHIFT
```

Comment:
For such customization it is better to use approach described here: #20869

Reply:
sure. Will fix it in next commit.